custom_components/powercalc/const.py

```diff
@@ -101,8 +101,14 @@
         "TRADFRI bulb E14 W op/ch 400lm": "LED1649C5",
         "TRADFRI bulb GU10 W 400lm": "LED1650R5",
         "TRADFRI bulb E27 WS opal 1000lm": "LED1732G11",
-        "TRADFRI bulb GU10 WW 400lm": "LED1837R5",
+        "TRADFRI bulb E14 WS opal 600lm": "LED1733G7",
+        "TRADFRI bulb E27 WS clear 806lm": "LED1736G9",
+        "TRADFRI bulb E14 WS opal 600lm": "LED1738G7",
+        "TRADFRI bulb E14 WS 470lm": "LED1835C6",
         "TRADFRI bulb E27 WW 806lm": "LED1836G9",
+        "TRADFRI bulb GU10 WW 400lm": "LED1837R5",
+        "TRADFRIbulbE14WSglobeopal470lm": "LED2002G5",
```

Comment:
This one does not seem right to me

Reply:
I know, but I took it from here:
![InkedScreenshot 2021-12-29 18 58 11_LI](https://user-images.githubusercontent.com/86839034/147690635-3268da86-0931-48c3-beb3-8e0911da5c98.jpg)

Reply:
Yes I see